pkg/openthread/include/platform_config.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) Baptiste Clenet
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     net
+ * @file
+ * @brief       Implementation of OpenThread platform config
+ *
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
+ * @}
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS
+ *
+ * The number of message buffers in buffer pool
+ */
+#if OPENTHREAD_ENABLE_NCP_UART
+#define OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS       5
```

Comment:
Put () around values

Reply:
This buffer size is so mall, which is the main reason why we cannot ping through the border router. I increased it to 40 and ping works well now :)

Reply:
The NCP doesn't build on samr21-xpro (not enough RAM) with this value, I decreased it to 30. The actual branch of this PR has been rebased on latest master.